source/ShoutrApp/src/app/peer-icon/peer-icon.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+
+@Component({
+  selector: 'zh-peer-icon',
+  templateUrl: './peer-icon.component.html',
+  styleUrls: ['./peer-icon.component.scss']
+})
+export class PeerIconComponent implements OnChanges{
+  perSide = 5
```

Comment:
Consider making this an `@Input` that defaults to 5

Reply:
done